build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -80,58 +80,58 @@
     </AndroidSdkItem>
   </ItemGroup>
   <ItemGroup>
-    <_NdkToolchain Include="arm-linux-androideabi-clang" Condition="$(AndroidSupportedTargetJitAbisForConditionalChecks.Contains(':armeabi:')) Or $(AndroidSupportedTargetJitAbisForConditionalChecks.Contains(':armeabi-v7a:'))">
+    <_NdkToolchain Include="arm-linux-androideabi-clang" Condition="$(AndroidSupportedTargetJitAbisForConditionalChecks.Contains(':armeabi:')) Or $(AndroidSupportedTargetJitAbisForConditionalChecks.Contains(':armeabi-v7a:')) Or $(AndroidSupportedTargetAotAbisForConditionalChecks.Contains (':win-armeabi:'))">
```

Comment:
This condition confuses me:

    Or $(AndroidSupportedTargetAotAbisForConditionalChecks.Contains (':win-armeabi:'))

What sets `win-armeabi`? The documentation for `$(AndroidSupportedTargetAotAbis)` doesn't mention any `win-`-prefixed values.

This could be fixed by just documenting these values in `Configuration.Override.props`.

I do wonder if this should be a `mxe-` prefix instead of a `win-` prefix, as is done for `$(AndroidSupportedHostJitAbis)`.